workplace-incoming-hook.go

```diff
@@ -360,7 +362,17 @@ func PushHandler(body string) {
 			// Third line (last commit message)
 			message += "```" + MessageEncode(lastCommit.Message) + "```"
 		}
-		SendWorkchatMessage(ThreadGitlab, message, ChatType)
+		// SendWorkchatMessage(ThreadGitlab, message, ChatType)
+		switch gr := helper.GroupDetection(j.Repository.URL); gr {
+		case "back-end":
+			SendWorkchatMessage(ThreadBackend, message, ChatType) // Change ThreadTuleap to your group Thread
```

Comment:
it's not allowed here bro, this repository must have general purpose only. or you can create new configuration for mapping thread in new configuration file.

Reply:
Ok , I will change it